packages/fxa-auth-server/lib/senders/emails/partials/automatedEmailResetPasswordTwoFactor/index.mjml

```diff
@@ -0,0 +1,17 @@
+<%# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
@chenba This is a new footer partial to reflect the Figma design - it could be used in your email PR too.